scripts/build

```diff
@@ -22,4 +22,4 @@ fi
 echo export GOPATH=$GOPATH
 
 mkdir -p bin
-go build -o bin/host-api
+GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o bin/host-api
```

Comment:
This forces the build to Linux which means if you are running on a Mac you could never run this.  Please remove this change.